File: datasets/IR-Plag/convertai-original-java-rest-kotlin/case-03/non-plagiarized/14/Soal3.kt
package IR

import java.text.DecimalFormat
import java.util.Scanner

/**
 *
 * @author 4900D0A19B6894A4A514E9FF3AFCC2C0
 */
fun main() {
    val sc = Scanner(System.`in`)
    print("Enter weight in pounds: ")
    val weight = sc.nextDouble()
    print("Enter feet: ")
    val feet = sc.nextDouble()
    print("Enter inches: ")
    val inches = sc.nextDouble()
    val height = feet * 12 + inches
    val bmi = weight * 0.45359237 / ((height * 0.0254) * (height * 0.0254))
    println("BMI is $bmi")
    when {
        bmi < 18.5 -> println("Underweight")
        bmi >= 18.5 && bmi < 25 -> println("Normal")
        bmi >= 25 && bmi < 35 -> println("Overweight")
        else -> println("Obese")
    }
}

File: datasets/IR-Plag/convertai-original-java-rest-python/case-03/non-plagiarized/14/Soal3.py
import math

print("Enter weight in pounds: ", end="")
weight = float(input())
print("Enter feet: ", end="")
feet = float(input())
print("Enter inches: ", end="")
inches = float(input())

height = feet * 12 + inches
bmi = weight * 0.45359237 / ((height * 0.0254) * (height * 0.0254))

print(f"BMI is {bmi:.2f}")

if bmi < 18.5:
    print("Underweight")
elif 18.5 <= bmi < 25:
    print("Normal")
elif 25 <= bmi < 35:
    print("Overweight")
else:
    print("Obese")

